src/carbon_calculator/tests/tests.py

```diff
@@ -183,6 +198,155 @@ def pretty_print_diffs(self, diffs, oldtime):
         else:
             print("carbon_calculator results consistent with input data from "+str(oldtime))
 
+    def test_info_events(self):
+        ''' Tests /cc/info/events url and returns a json of events. '''
+
+        event_list = self.client.get("/cc/info/events", {}) # status code = 200
+        event_list_json = jsons.loads(event_list.content)   # loads into json
+        #print(event_list_json)                             # uncomment this to see json
```

Comment:
So i see a bunch of these commented out print statements.  Ideally, we don't really want "dead code" in there.  If its not a comment explaining something / some complex code we don't want to rewrite then it should probably not be there.  

One trick we could use here is that, just like we have DEBUG=True/False in settings.py, you could declare a similar constant in this file (maybe called `DEBUG=True` and then put all these print statements under an if statement that checks if this DEBUG=True then it prints else it leaves it.  So you won't need to uncomment anything if you wanted those print statements.  You can control it with just one change.